content/200-concepts/100-components/01-prisma-schema/09-indexes.mdx

```diff
@@ -24,13 +24,22 @@ If you are upgrading from a version earlier than 4.0.0, these changes to index c
 
 You can configure indexes, unique constraints, and primary key constraints with the following attribute arguments:
 
-- The `length` argument is available **in MySQL only** on the `@id`, `@@id`, `@unique`, `@@unique` and `@@index` attributes in version `3.5.0` and later. It allows Prisma to support indexes and constraints on `String` and `Bytes` types.
+- The `length` argument allows you to specify a maximum length for the subpart of the value to be indexed on `String` and `Bytes` types
 
-- The `sort` argument is available for all databases on the `@unique`, `@@unique` and `@@index` attributes in version `3.5.0` and later. Additionally, SQL Server also allows it on `@id` and `@@id`.
+  - Available on the `@id`, `@@id`, `@unique`, `@@unique` and `@@index` attributes **in MySQL only**, in versions 3.5.0 and later
 
-- The `type` argument is available **in PostgreSQL only** on the `@@index` attribute in version `3.6.0` and later. This argument allows Prisma to support the `Hash` index access method as well as the default `BTree` access method. In version `3.14.0` and later the `Gist`, `Gin`, `SpGist` and `Brin` access methods are also available.
+- The `sort` argument allows you to specify the order that the entries of the constraint or index are stored in the database
 
-- The `clustered` argument is available **in SQL Server only** on the `@id`, `@@id`, `@unique`, `@@unique` and `@@index` attribute in version `3.13.0` and later. It allows Prisma to configure (non)clustered indexes.
+  - Available on the `@unique`, `@@unique` and `@@index` attributes in all databases and on the `@id` and `@@id` attributes in SQL Server, in versions 3.5.0 and later
+
+- The `type` argument allows you to support index access methods other than PostgreSQL's default `BTree` access method
+
+  - Available on the `@@index` attribute **in PostgreSQL only**
+  - The `Hash` index access method is available in versions 3.6.0 and later
+  - Additionally the `Gist`, `Gin`, `SpGist` and `Brin` access methods are available in versions 3.14.0 and later
```

Comment:
Similar to something I commented in Slack:
I think the version information really does not need to be present here, and can be part of the detailled section below.